src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationPredictedVsObservedChartPresenter.cs

```diff
@@ -61,6 +64,7 @@ protected override void ClearChartAndDataRepositories()
       {
          base.ClearChartAndDataRepositories();
          _identityRepositories.Clear();
+         _deviationLineRepositories.Clear();
```

Comment:
do we nee dot have thow two repositories? Isn't identity just a specific deviation? I am not sure. Just asking